azkaban-web-server/src/main/java/azkaban/imagemgmt/servlets/ImageTypeServlet.java

```diff
@@ -175,29 +175,41 @@ protected void handlePost(
         throw new ImageMgmtValidationException(ErrorCode.BAD_REQUEST, "Required field imageType is"
             + " null. Must provide valid imageType to create/register image type.");
       }
-      if (!hasImageManagementPermission(imageType, session.getUser(), Type.CREATE)) {
-        log.debug(String.format("Invalid permission to create image type for "
-            + "user: %s, image type: %s.", session.getUser().getUserId(), imageType));
-        throw new ImageMgmtInvalidPermissionException(ErrorCode.FORBIDDEN, "Invalid permission to "
-            + "create image type");
+      final Integer imageTypeId;
+      genericImageType.setModifiedBy(session.getUser().getUserId());
+      genericImageType.setCreatedBy(session.getUser().getUserId());
```

Comment:
I thought createdBy and modifiedBy are different people. You might only need to set modifiedBy.

Reply:
in the case of creating the image_types table maybe, but here either the person a create is happening where both will be the same. or a new entry in the image_ownership table takes place for which both created and modified entry will be the user logged in.